bigbluebutton-html5/imports/ui/components/nav-bar/component.jsx

```diff
@@ -80,6 +89,12 @@ class NavBar extends Component {
     }
   }
 
+  componentDidUpdate(prevProps, prevState) {
+    if (JSON.stringify(prevProps.activeChats) !== JSON.stringify(this.props.activeChats)) {
```

Comment:
In other spots we use `_.isEqual(a, b)` for similar comparisons. I am not 100% sure whether it does more than stringify compare. Have you considered lodash isEqual comparison?